app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -71,20 +71,28 @@
 
```

Comment:
This seems to have unrelated changes

Reply:
@ashishkumar0207 This change was needed since i need to add captions to the Contribution constructor

Reply:
In ContributionsFragment ?

Reply:
No I mean I have added captions to the Contribution Constructor. And here we are initialising the constructor so we i need to add some value for caption

Reply:
Yes, my comment is regarding the ContributionsFragment, but seems like you are referring to ContrubutionDao